### DISCUSSION TITLE
`trace_replayBlockTransactions` fails for certain blocks

### DISCUSSION THREAD

**Discussion by lianqiao** [erigontech/erigon#3432] [Q&A]
The code works for almost all blocks, but when I replay to very low number blocks (the blocks which were mined in 2016), it takes very long time (like ~1 minute vs 0.1 seconds) to repsond an error message.

For exmaple, block 2287736 responses:
```
{'jsonrpc': '2.0',
 'id': 0,
 'error': {'code': -32000,
  'message': 'first run for txIndex 1 error: nonce too high: address 0x3Ad6beC12af7b7f295380298a0664a780c87c163, tx: 6 state: 0'}}
```

The code:
```
import requests, json, pprint, time, sys, os
def rpc_call(method, params):
    payload = {
        "method": method,
        "params": params,
        "jsonrpc": "2.0",
        "id": 0
    }
    return requests.post(
          'http://localhost:8545',
          data=json.dumps(payload),
          headers={"content-type": "application/json"}).json()

rpc_call('trace_replayBlockTransactions', ['0x%x' % 2287736, ['stateDiff']])
```

**Comment by lianqiao** [comment]
I tried this block on openethereum node. It returns a response. But very very slow. It took 98 seconds vs 0.03 second for another regular block. 

Is there anything wrong in this block? I am afraid the node did some mining work when processing the replay request.

**Reply by AskAlexSharov** [reply]
Shanghai attack

**Reply by lianqiao** [reply]
Is there a way to process more internal transactions instead of returning an error? The error message looks kind of timeout / out of perf range for me.
